test/standard_test.js

```diff
@@ -1,20 +0,0 @@
-var standard = require('mocha-standard')
```

Comment:
but sure, i don't really care much about running lint testing often anyway, if this is a bit more comfy for you guys go for it